pkg/api/validation/validation.go

```diff
@@ -1934,6 +1935,16 @@ func ValidateSeccompPodAnnotations(annotations map[string]string, fldPath *field
 	return allErrs
 }
 
+func validateAppArmorProfile(p string, fldPath *field.Path) field.ErrorList {
+	if p == "docker/default" {
```

Comment:
Yes, thanks for pointing this out, I meant to highlight it in the proposal as well. I had some internal debate with myself, so I'm curious to hear another perspective. My logic for going with `docker/default` was in the interest of explicitly stating the expectations. It will probably be the case (once other runtimes have AppArmor support) that the default behavior will be different. For instance, if I deploy an app with `runtime/default` expecting the docker profiles to a cluster running rkt, it might run unconfined without my knowledge. If I need to explicitly state the runtime default to use, the container would be rejected if a different runtime was in use.